main.go

```diff
@@ -1449,6 +1449,50 @@ func (tf *transformer) transformGo(file *ast.File) *ast.File {
 	// and that's not allowed in the runtime itself.
 	if flagLiterals && curPkg.ToObfuscate {
 		file = literals.Obfuscate(file, tf.info, fset, tf.linkerVariableStrings)
+
+		// some imported constants might not be needed anymore, remove unnecessary imports
+		usedImports := make(map[string]bool)
+		ast.Inspect(file, func(n ast.Node) bool {
+			node, ok := n.(*ast.Ident)
+			if !ok {
+				return true
+			}
+
+			uses, ok := tf.info.Uses[node].(*types.PkgName)
+			if !ok {
+				return true
+			}
+
+			usedImports[uses.Imported().Path()] = true
+
+			return true
+		})
+
+		log.Printf("used imports, %+v", usedImports)
```

Comment:
delete? if we want to keep it it should be debugf, but I'm not sure how useful that will be to users of `garble -debug`